README.md

```diff
@@ -150,6 +150,17 @@ being:
 - `i + 1` replaces `1,1,1,1,1` into `1,2,3,4,5`, assuming that each selection is
   on a different digit.
 
+### Status bar
+
+Dance provides several status bar segments (left-aligned) exposing info similar
+to Kakoune's default mode-line:
+
+- current mode: click to switch to another mode
+- macro recording status: click to stop recording
+- current count prefix: click to reset to 0
+- current register: click to unset
+- dance error: click to copy the last one
```

Comment:
nit:
```suggestion
- dance error: click to copy the full description of the last error
```

(Or something else that implies that the full error is copied, not just what's shown on the status bar, which is truncated)